### PR TITLE
es6-template-literals - Aaron Soukaphay

### PR DESCRIPTION
**Pull request by aaronsoukaphay**
Screenshot 2023-08-07 at 10 48 35 AM

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Javascript ES6 - Template Literals</title>
+  </head>
+  <body>
+    <script>
+      const number = 9200;
+      const street = 'Irvine Center Drive';
+      const address = number + ' ' + street;
```
Comment (shawnkost) [inline es6-template-literals/index.html]: This should be a template literal

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      const templateAddress = `LearningFuze: ${number} ${street}`;
```
Comment (shawnkost) [inline es6-template-literals/index.html]: Delete this

File: es6-template-literals/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+      console.log('LearningFuze:', address);
+      console.log(templateAddress);
```
Comment (shawnkost) [inline es6-template-literals/index.html]: Delete this

File: es6-template-literals/index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      const age = 25;
+
+      const bio =
+        'My name is ' +
+        firstName +
+        ' ' +
+        lastName +
+        ' and I am ' +
+        age +
+        ' years old.';
```
Comment (shawnkost) [inline es6-template-literals/index.html]: This should be a template literal

File: es6-template-literals/index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      const templateBio = `Bio: My name is ${firstName} ${lastName} and I am ${age} years old.`;
```
Comment (shawnkost) [inline es6-template-literals/index.html]: Delete this

File: es6-template-literals/index.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+      console.log('Bio:', bio);
+      console.log(templateBio);
```
Comment (shawnkost) [inline es6-template-literals/index.html]: Delete this

File: es6-template-literals/index.html [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+      const fruit = 'apple';
+      const item = 'pen';
+
+      const applePen = fruit + item;
```
Comment (shawnkost) [inline es6-template-literals/index.html]: This should be a template literal

File: es6-template-literals/index.html [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+      const templateApplePen = `${fruit + item}`;
```
Comment (shawnkost) [inline es6-template-literals/index.html]: Delete this

File: es6-template-literals/index.html [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+      console.log('I have a', item);
+      const templateItem = `I have a ${item}`;
+
+      console.log('I have an', fruit);
+      const templateFruit = `I have an ${fruit}`;
+
+      console.log(templateItem);
+      console.log(templateFruit);
+      console.log(applePen);
+      console.log(templateApplePen);
```
Comment (shawnkost) [inline es6-template-literals/index.html]: Delete this

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Aaron! You are not supposed to be creating new variables, you have all the variables you need. You should be changing them to use template literals rather than string concatenation

**Comment by aaronsoukaphay** [comment]
Screenshot 2023-08-07 at 11 02 02 AM

File: es6-template-literals/index.html [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+      const age = 25;
+
+      const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+      console.log('Bio:', bio);
+
+      const fruit = 'apple';
+      const item = 'pen';
+
+      const applePen = `${fruit + item}`;
```
Comment (shawnkost) [inline es6-template-literals/index.html]: No `+` needed. That's still doing string concatenation.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Aaron!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**